src/main.ts

```diff
@@ -43,10 +43,10 @@ async function main(argv, env): Promise<void> {
                 workflowLogger.info(event.data)
             }
         })
-        eventSource.addEventListener('error', (errorEvent) => {
+        eventSource.addEventListener('error', function (event) {
```

Comment:
why? 
and if we are changing it here lets change it in the other eventListeners